src/Movies/Movies.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Movie from'../Movie/Movie';
+import '../Movies/Movies.css'
+
+const Movies = ({movies}) => {
+    const movieCards = movies.map(movie => {
+        return (
+            <Movie
+             title={movie.title}
+             image={movie.poster_path}
+             imageBackdrop={movie.backdrop_path}
+             rating={movie.average_rating}
+             releaseDate={movie.release_date}
+             id={movie.id}
+             key={movie.id}
```

Comment:
Adding a unique key for React was a good way to render the movie. I'm curious what would happen if we tried Date.now()